src/main/handlebars/feed.handlebars

```diff
@@ -40,9 +40,15 @@
           </div>
           <div class="images size-{{size images}}">
             {{#each images}}
-              <a href="/image/{{slug}}/full-{{.}}.webp" onclick="return lightbox.show(this)">
-                <img {{#unless ../@first}}loading="lazy"{{/unless}} src="/image/{{slug}}/thumb-{{.}}.webp" width="1024">
-              </a>
+              {{#if is.video}}
+                <video controls preload="none" width="100%" poster="/image/{{slug}}/screen-{{name}}.webp">
+                  <source src="/image/{{slug}}/video-{{name}}.mp4" type="video/mp4">
+                </video>
+              {{else}}
+                <a href="/image/{{slug}}/full-{{.}}.webp" onclick="return lightbox.show(this)">
```

Comment:
We use `{{.}}` here for BC reasons, once all data has been convertd, we can use `{{name}}` here.